pkg/service/roomallocator.go

```diff
@@ -154,6 +159,10 @@ func (r *StandardRoomAllocator) CreateRoom(ctx context.Context, req *livekit.Cre
 }
 
 func (r *StandardRoomAllocator) ValidateCreateRoom(ctx context.Context, roomName livekit.RoomName) error {
+	if len(roomName) > r.config.Room.MaxRoomNameLength {
+		return fmt.Errorf("%w: max length %d", ErrRoomNameExceedsLimits, r.config.Room.MaxRoomNameLength)
```

Comment:
Should these check if the config is non-zero? Did not see a default config value for these.